packages/ndla-ui/src/CampaignBlock/CampaignBlock.tsx

```diff
@@ -67,42 +67,33 @@ const Container = styled("div", {
         },
       },
       right: {
+        display: "flex",
+        flexDirection: "column-reverse",
         "@/tablet": {
+          display: "grid",
           gridTemplateColumns: "auto minmax(230px, 455px)", //required for campaign block in myNdla
         },
         "@supports not (container-type: inline-size)": {
+          display: "grid",
```

Comment:
Hvorfor trenger vi plutselig dette?

Reply:
For kampanjeblokk med bilde på høyre side skal ha bilde øverst på små enheter akkurat som den andre varianten. Da måtte jeg få inn `display:flex` for å få reversert kolonnen også må jeg sette display tilbake til grid på større skjermer

Reply:
Det er jo egentlig litt dumt. Å bruke `-reverse` er nesten alltid et UU-brudd, med mindre bildet er dekorativt.

Reply:
Regner med at disse bildene er dekorative? 😅 Men da må vi evt droppe bilde på høyre helt

Reply:
Vi setter alt på de, så de er vel per def ikke dekorative